agent/agent.go

```diff
@@ -586,7 +586,7 @@ func (a *Agent) runTaskTimeoutCommands(ctx context.Context, tc *taskContext) {
 		return
 	}
 	if taskGroup.Timeout != nil {
-		err := a.runCommands(ctx, tc, taskGroup.Timeout.List(), runCommandsOptions{})
+		err := a.runCommands(ctx, tc, taskGroup.Timeout.List(), runCommandsOptions{}, "task-timeout")
```

Comment:
Can we make global constants for these and group them in the same place?

Reply:
smart, done